### PR TITLE
Field Description CSVs

### PR DESCRIPTION
**Pull request by seemantk**
This is an initial effort at creating downloadable field description files.  Here, we're using Jekyll to create CSV files for disclosures, other restrictions and public-financing, using the data in jekyll configuration.

### CODE REVIEW

File: downloads/metadata/disclosure-fields.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+---
+Field Name,Question,Short Label,Additional Notes{% for control in site.data.sections.disclosure.controls %}
+"{{ control.name }}","{{ control.question }}","{{ control.label }}","{{ control.note }}"{% endfor %}
```
Comment (curran) [inline downloads/metadata/disclosure-fields.csv]: This is pretty cool how this template turned out!

File: downloads/metadata/public-financing-fields.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+---
+Field Name,Question,Short Label,Additional Notes{% for control in site.data.sections.["public-financing"].controls %}
```
Comment (curran) [inline downloads/metadata/public-financing-fields.csv]: I'm seeing a Jekyll error from this line:

```
    Liquid Warning: Liquid syntax error (line 1): Expected id but found open_square in "control in site.data.sections.["public-financing"].controls" in downloads/metadata/public-financing-fields.csv
```

Comment (seemantk) [inline downloads/metadata/public-financing-fields.csv]: Are there any weird characters in _data/sections/public-financing/controls.yml by any chance?

Comment (curran) [inline downloads/metadata/public-financing-fields.csv]: Not that I can see.

I just tried removing the period and that got rid of the warning.

`sections.["public-financing"]` Warning
`sections["public-financing"]` No warning

Comment (seemantk) [inline downloads/metadata/public-financing-fields.csv]: nice catch.

**Comment by curran** [comment]
Lots of changes to `master` since this started, I think I'll just sync them up real quick.

**Comment by curran** [comment]
@seemantk I believe these recent changes wrap up this PR. Please have a look and merge at your discretion.

Now there is a file `_site/downloads/cfi-laws-database.zip`, with the following layout:

```
├── cfi-laws-database
│   ├── LICENSE
│   ├── data
│   │   ├── Laws_00_IdentifierTable.csv
│   │   ├── Laws_01_Defintions.csv
│   │   ├── Laws_02_Contributions_1.csv
│   │   ├── Laws_02_Contributions_2.csv
│   │   ├── Laws_02_Contributions_3.csv
│   │   ├── Laws_03_Disclosure_1.csv
│   │   ├── Laws_03_Disclosure_2.csv
│   │   ├── Laws_03_Disclosure_3.csv
│   │   ├── Laws_04_PublicFinancing.csv
│   │   └── Laws_05_Other.csv
│   └── metadata
│       ├── disclosure-fields.csv
│       ├── other-restrictions-fields.csv
│       └── public-financing-fields.csv
```

**Comment by curran** [comment]
@seemantk I believe these recent changes wrap up this PR. Please have a look and merge at your discretion.

Now there is a file `_site/downloads/cfi-laws-database.zip`, with the following layout:

```
├── cfi-laws-database
    ├── LICENSE
    ├── data
    │   ├── Laws_00_IdentifierTable.csv
    │   ├── Laws_01_Defintions.csv
    │   ├── Laws_02_Contributions_1.csv
    │   ├── Laws_02_Contributions_2.csv
    │   ├── Laws_02_Contributions_3.csv
    │   ├── Laws_03_Disclosure_1.csv
    │   ├── Laws_03_Disclosure_2.csv
    │   ├── Laws_03_Disclosure_3.csv
    │   ├── Laws_04_PublicFinancing.csv
    │   └── Laws_05_Other.csv
    └── metadata
        ├── disclosure-fields.csv
        ├── other-restrictions-fields.csv
        └── public-financing-fields.csv
```

This takes the place of the two separate .zip files we had before.

File: README.md [hunk 3]
```diff
@@ -57,24 +57,13 @@ Exporting Laws_04_PublicFinancing to CSV...
...
-mv ~/Downloads/Field\ Names-Descriptions\ for\ Visuals_v5.xlsx\ -\ Other\ Restrictions.csv ./other-restrictions-fields.csv 
-```
-
-Note: Make sure there are no empty rows in the Public Funding fields sheet. If there are empty lines in the CSV file, they should be deleted.
+Field descriptions live inside the Jekyll configurations, under `_data/sections/<section>/controls.yml`. These files are automatically converted to .CSV files when you run `./bin/package-downloads.sh`.
 
 ### Packaging Download Files
 
-The database content as well as metadata (field descriptions and about button contents) is packaged into .zip files for download. To create these .zip files, run the following script.
+The database content as well as metadata (field descriptions and about button contents) is packaged into a .zip file for download. To create this .zip file, run the following script:
```
Comment (seemantk) [inline README.md]: Aha, sorry -- the about button contents are not packaged in the zip.

**Pull request merged**